README.md

```diff
@@ -143,7 +143,7 @@ It supports backpressure.
 ### instance.publish(packet, done)
 
 Publish the given packet to subscribed clients and functions. A packet
-must be valid for [mqtt-packet](http://npm.im/mqtt-packet).
+must be valid for [aedes-packet](http://npm.im/aedes-packet).
```

Comment:
In aedes-packet, everything is defaulted:

https://github.com/mcollina/aedes-packet/blob/master/packet.js#L4-L11.

I think a much better definition can be: a packet must have the following properties...